commander/txs_tracking_test.go

```diff
@@ -182,21 +182,25 @@ func (s *TxsTrackingTestSuite) runInTransaction(
 	handler(txStorage, txsCtx)
 }
 
-func (s *TxsTrackingTestSuite) startWorkers() {
-	s.cmd.startWorker("Test Txs Requests Sending", func() error {
-		err := s.cmd.startSendingRequestedTxs()
+func (s *TxsTrackingTestSuite) startWorkers(wantTrackingError bool) {
+	s.cmd.startWorker("Test Sending Requested Txs", func() error {
+		err := s.cmd.sendRequestedTxs()
 		s.NoError(err)
-		return nil
+		return err
 	})
-	s.cmd.startWorker("Test Failed Txs Tracking", func() error {
-		err := s.cmd.startTrackingSentTxs()
-		s.NoError(err)
-		return nil
+	s.cmd.startWorker("Test Tracking Sent Txs", func() error {
+		err := s.cmd.trackSentTxs()
+		if wantTrackingError {
+			s.Error(err)
+		} else {
+			s.NoError(err)
+		}
```

Comment:
As I can see wantTrackingError is always true, remove function argument and if statement.